neutrino.go

```diff
@@ -697,6 +704,7 @@ func NewChainService(cfg Config) (*ChainService, error) {
 		blocksOnly:        cfg.BlocksOnly,
 		mempool:           NewMempool(),
 		proxy:             cfg.Proxy,
+		persistToDisk:     cfg.PersistToDisk,
```

Comment:
When rebase, we should stay consistent with upstream and have `persistToDisk:` and then `broadcastTimeout:`.